source/adios2/core/Operator.h

```diff
@@ -37,7 +37,6 @@ class Operator
         COMPRESS_SZ = 6,
         COMPRESS_ZFP = 7,
         COMPRESS_MGARDPLUS = 8,
-        COMPRESS_BLOSC2 = 9,
```

Comment:
I thought we were removing `Blosc` but leaving `Blosc2` so why is this one being deleted?